integration-tests/tests/common/mod.rs

```diff
@@ -78,6 +81,20 @@ impl IntegrationTest {
     ) {
         let seconds = Duration::from_secs(timeout);
 
+        #[cfg(any(has_sev, has_kvm))]
+        let mut cmd = Command::new(self.keep)
+            .current_dir(self.root)
+            .arg("exec")
```

Comment:
Maybe we could have a variable for the first `.arg()` that changes based on whether it's SGX? All the other 10 lines are the same.

Reply:
the special SGX case will go away, once it is integrated in `enarx-keep`

Reply:
Ah, fine, then it's good the way it is.